src/components/MediaSettings/MediaSettings.vue

```diff
@@ -215,13 +214,11 @@ import { useDevices } from '../../composables/useDevices.js'
 import { useIsInCall } from '../../composables/useIsInCall.js'
 import { AVATAR, CALL, PARTICIPANT, VIRTUAL_BACKGROUND } from '../../constants.js'
 import BrowserStorage from '../../services/BrowserStorage.js'
+import { getTalkConfig } from '../../services/CapabilitiesManager.ts'
 import { useGuestNameStore } from '../../stores/guestName.js'
 import { useSettingsStore } from '../../stores/settings.js'
 import { localMediaModel } from '../../utils/webrtc/index.js'
 
-const recordingEnabled = getCapabilities()?.spreed?.config?.call?.recording || false
-const recordingConsent = getCapabilities()?.spreed?.config?.call?.['recording-consent']
```

Comment:
one is per server level and one per conversation afaik ? (static and dynamic)

Reply:
`recordingEnabled` - if recording server is available (should be from remote)
`recordingConsent` - if consent is enabled by administrator (should be from remote)
`conversation.recordingConsent` - if enabled per conversation